src/database/index.tsx

```diff
@@ -192,6 +192,8 @@ export const createBaselineScenario = async () => {
     baselineScenarioRef = await db.collection(scenariosCollectionId).add({
       name: "Baseline Scenario",
       baseline: true,
+      dataSharing: false,
+      dailyReports: false,
```

Comment:
Thanks for adding these.  I've manually updated the existing scenarios in Firebase to set these values